gammapy/irf/psf_map.py

```diff
@@ -299,6 +299,48 @@ def from_energy_dependent_table_psf(cls, table_psf):
         exposure_map.quantity += data
         return cls(psf_map=psf_map, exposure_map=exposure_map)
 
+    @classmethod
+    def from_gauss(cls, energy, rad, sigma):
```

Comment:
We could directly use a `MapAxis` object here for `energy` and `rad` (in this case I would rename it to `energy_axis_true` and `rad_axis` to be compatible with `MapDataset.create()`. For "historical" reasons the `MapAxis` has not really been used for IRFs, but as the `.from_gauss()` method is part of the `PSFMap`API (where we rely on `Map` and `MapAxis` objects) this seems more natural...

Reply:
I think you're right. I did it this way because the input to the `EnergyDependentTablePSF` were not `MapAxis` but just 1-d arrays. I changed it now.